spec/comment_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { create(:user) }
+  let(:post) { create(:post, author: user) }
+
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User') }
+    it { should belong_to(:post).class_name('Post') }
+  end
+end
```

Comment:
Nice job so far. However, according to the project requirements, any custom methods in the models should have a corresponding unit test. Kindly consider adding unit tests to the `increment_comments_count` method in the comment model here. :pray: